packages/zowe-explorer/webpack.config.js

```diff
@@ -16,6 +16,7 @@
 const path = require("path");
 var webpack = require("webpack");
 var fs = require("fs");
+const { features } = require("process");
```

Comment:
@lauren-li this line is what I was referring to and it can be taken out.  I didn't realize it was there on the commit until I looked after because it was automatically added when I was trying to add the exclusion.